doc/design/vlog_guide.md

```diff
@@ -0,0 +1,13 @@
+This is a follow up for switching logging to VLOG. I feel we need a VLOG level guide for the PaddlePaddle developers. Here is what I have in mind
```

Comment:
We are going to follow the convention adopted by Chromium to [use `VLOG` instead of `LOG(INFO)` to generate *debug information*](https://groups.google.com/a/chromium.org/d/msg/chromium-dev/3NDNd1KzXeY/AZKMMx37fdQJ).   Please be aware that usually log messages are still via `LOG`.

For more information about `VLOG` please refer to the [official document of glog](http://htmlpreview.github.io/?https://github.com/google/glog/blob/master/doc/glog.html#verbose).

`VLOG` requires a *verbose level* parameter, like that `LOG` requires a `severity level`.  This document explains the verbose level value for various kinds of VLOG messages.